grapher/core/LegacyToOwidTable.ts

```diff
@@ -393,8 +393,8 @@ const fullJoinTables = (
         // When we get a mergeFallbackLookupColumn then it can happen that a table does not have all the
         // columns of the main index. In this case, just return an empty map because that will lead all
         // lookups by main index to fail and we'll try the fallback index
-        !mergeFallbackLookupColumns ||
-        difference(indexColumnNames, table.columnSlugs).length === 0
+        mergeFallbackLookupColumns &&
+        difference(indexColumnNames, table.columnSlugs).length > 0
             ? table.rowIndex(indexColumnNames)
             : new Map()
```

Comment:
You will also have to switch these two up.

Reply:
🤦of course